pkg/controllers/kube-apiserver.go

```diff
@@ -255,45 +261,19 @@ func (s *KubeAPIServer) configure(cfg *config.Config) error {
 	return nil
 }
 
-func (s *KubeAPIServer) configureAuditPolicy() error {
-	data := []byte(`
-apiVersion: audit.k8s.io/v1
-kind: Policy
-metadata:
-  name: Default
-# Don't generate audit events for all requests in RequestReceived stage.
-omitStages:
-- "RequestReceived"
-rules:
-# Don't log requests for events
-- level: None
-  resources:
-  - group: ""
-    resources: ["events"]
-# Don't log oauth tokens as metadata.name is the secret
-- level: None
-  resources:
-  - group: "oauth.openshift.io"
-    resources: ["oauthaccesstokens", "oauthauthorizetokens"]
-# Don't log authenticated requests to certain non-resource URL paths.
-- level: None
-  userGroups: ["system:authenticated", "system:unauthenticated"]
-  nonResourceURLs:
-  - "/api*" # Wildcard matching.
-  - "/version"
-  - "/healthz"
-  - "/readyz"
-# A catch-all rule to log all other requests at the Metadata level.
-- level: Metadata
-  # Long-running requests like watches that fall under this rule will not
-  # generate an audit event in RequestReceived.
-  omitStages:
-  - "RequestReceived"`)
-
+func (s *KubeAPIServer) configureAuditPolicy(cfg *config.Config) error {
+	p, err := apiserver.GetPolicy(cfg.ApiServer.AuditLog.Profile)
```

Comment:
Would it be possible to validate this in the config package?

Reply:
Yes, but I'm doubtful of the value there. Is the idea to catch breaking changes to the OpenShift policies?

Reply:
You would get the validation also in `show-config` command, so that any invalid values also signal the error and you dont need to wait until starting microshift.

Reply:
Ah my bad, I see what you mean (thought you were referring to testing).

Reply:
I think this is not addressed yet? Otherwise the PR looks pretty good